nipype/utils/filemanip.py

```diff
@@ -699,37 +699,49 @@ def loadpkl(infile):
     infile = Path(infile)
     fmlogger.debug('Loading pkl: %s', infile)
     pklopen = gzip.open if infile.suffix == '.pklz' else open
+
+    with SoftFileLock('%s.lock' % infile.name):
+        with pklopen(str(infile), 'rb') as pkl_file:
+            pkl_contents = pkl_file.read()
```

Comment:
```suggestion
        pkl_contents = infile.read_bytes()
```
although we can wait until only Python +3.5 is supported.

Reply:
Yeah, I feel like we've been bitten by trying to jump ahead of ourselves. I'd rather stick with idioms universal to our support window.

Reply:
Oh, and actually that wouldn't work, since we need to use `gzip.open` for `pklz` files anyway.